test/infrastructure/docker/go.mod

```diff
@@ -3,28 +3,24 @@ module sigs.k8s.io/cluster-api/test/infrastructure/docker
 go 1.12
 
 require (
-	cloud.google.com/go v0.38.0 // indirect
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-logr/logr v0.1.0
-	github.com/gogo/protobuf v1.2.2-0.20190723190241-65acae22fc9d // indirect
 	github.com/onsi/ginkgo v1.10.3
 	github.com/onsi/gomega v1.7.1
 	github.com/pkg/errors v0.8.1
-	github.com/sirupsen/logrus v1.4.2 // indirect
 	go.uber.org/zap v1.10.0 // indirect
-	golang.org/x/net v0.0.0-20190813141303-74dc4d7220e7 // indirect
-	k8s.io/api v0.0.0-20190918195907-bd6ac527cfd2
-	k8s.io/apimachinery v0.0.0-20190817020851-f2f3a405f61d
+	k8s.io/api v0.0.0-20191121015604-11707872ac1c
+	k8s.io/apimachinery v0.0.0-20191121015412-41065c7a8c2a
 	k8s.io/client-go v11.0.0+incompatible
```

Comment:
Could we just use `k8s.io/client-go v0.0.0-20190918160344-1fbdaa4c8d90` here? Instead of having the replace directive, that is. I'm not sure what the original reason was in having the replace directive.

Reply:
Unfortunately not - it's forced to this by the `go` toolchain. This specific version comes from our import of kind, iirc.